README.md

```diff
@@ -42,91 +62,113 @@ Notice that what we see in our the view source tab looks very similar to HTML yo
 
 ```
 
-It may look like our browser is simply displaying the HTML that we see, but it is not.  Instead, our browser is directly displaying the Document Object Model of the respective webpage. What does that mean?  What is the Document Object Model, and how is it different than the HTML for the page?
+It may look like our browser is displaying the HTML that we see, but it is not.
+Instead, our browser is displaying the Document Object Model of the respective
```

Comment:
Yeah, I think I feel a bit betrayed as a reader here like...wait, you just said 

> The browser interprets this HTML, along with CSS and JavaScript, to present the source code the way it appears in the browser.

> It may look like our browser is displaying the HTML that we see, but it is not.

Interprets versus look is kinda subtle there.